nullaway/src/test/java/com/uber/nullaway/NullAwayJSpecifyGenericsTests.java

```diff
@@ -1494,6 +1494,34 @@ public void interactionWithContracts() {
         .doTest();
   }
 
+  @Test
+  public void testForStaticMethodCallAsAParam() {
```

Comment:
@akulk022 have you confirmed that this test fails with two unexpected errors if you remove the fixes in GenericsChecks?

Reply:
Yes, I verified it for both scenarios by removing the fix and seeing if the test case fails with the same exception as before.